Lib/pickletools.py

```diff
@@ -1748,7 +1764,12 @@ def __init__(self, name, code, arg,
       stack_before=[anyobject],
       stack_after=[],
       proto=0,
-      doc="Discard the top stack item, shrinking the stack by one item."),
+      doc="""Discard the top stack item, shrinking the stack by one item.
+
+      If the stack has no items in it and the metastack is not empty, then
+      this opcode will act like POP_MARK and pop the top of the metastack
+      into the current stack.
```

Comment:
"Metastack" is an implementation detail. The behavior is equivalent of popping markobject from the stack (and it was implemented so initially). So no additional explanation is needed.